dask_cuda/tests/test_explicit_comms.py

```diff
@@ -4,6 +4,9 @@
 import signal
 import time
 from functools import partial
+import signal
```

Comment:
@younseojava  Why are these includes necessary? Seem not to be required `rapidsai/dask_cuda`.